apple-nav/src/comp/navwrapper/topnav/navigationData.js

```diff
@@ -0,0 +1,40 @@
+export const navLinks = [
```

Comment:
I really appreciate that you took the time to decouple this information from any component files.